cmd/generate/config/rules/settlemint.go

```diff
@@ -0,0 +1,58 @@
+package rules
+
+import (
+	"github.com/zricethezav/gitleaks/v8/cmd/generate/config/utils"
+	"github.com/zricethezav/gitleaks/v8/cmd/generate/secrets"
+	"github.com/zricethezav/gitleaks/v8/config"
+)
+
+func SettlemintPersonalAccessToken() *config.Rule {
+	// define rule
+	r := config.Rule{
+		Description: "Found a Settlemint Personal Access Token.",
+		RuleID:      "settlemint-personal-access-token",
+		Regex:       utils.GenerateUniqueTokenRegex(`(sm_pat_)[a-zA-Z0-9]+`, false),
```

Comment:
Do the tokens have a fixed or minimum length? This will match `sm_pat_a` which likely isn't a real token.

Reply:
ah yes, looks like the tokens I was able to generate had a length of 12 characters after the prefix. Added this. 👍